### ISSUE TITLE
Build error: RiderSdkPackageVersion.props not found

### ISSUE THREAD

**Issue by TimSirmovics** [JetBrains/ForTea#13]
I downloaded a copy of the fork and tried to execute Backend\build.ps1 to build for R# but got the following error:

> PS D:\Development\GitHub\JetBrains\ForTea-net193\Backend> ./build
> Windows PowerShell 5.1.18362.145
> Microsoft (R) .NET Core SDK version 3.0.100
> Microsoft (R) Build Engine version 16.3.0+0f4c62fea for .NET Core
> Copyright (C) Microsoft Corporation. All rights reserved.
> 
>   Restore completed in 1.53 sec for D:\Development\GitHub\JetBrains\ForTea-net193\Backend\build\_build.csproj.
> D:\Development\GitHub\JetBrains\ForTea-net193\Backend\Directory.Build.props(5,3): error MSB4019: The imported project "D:\Development\GitHub\JetBrains\ForTea-net193\Backend\RiderSdkPackageVersion.props" was not found. Confirm that the path in the <Import> declaration is correct, and that the file exists on disk. [D:\Development\GitHub\JetBrains\ForTea-net193\Backend\build\_build.csproj]
> 
> Build FAILED.
> 
> D:\Development\GitHub\JetBrains\ForTea-net193\Backend\Directory.Build.props(5,3): error MSB4019: The imported project "D:\Development\GitHub\JetBrains\ForTea-net193\Backend\RiderSdkPackageVersion.props" was not found. Confirm that the path in the <Import> declaration is correct, and that the file exists on disk. [D:\Development\GitHub\JetBrains\ForTea-net193\Backend\build\_build.csproj]
>     0 Warning(s)
>     1 Error(s)

**Comment by kirillgla** [comment]
Thank you for your report! Indeed, the build instructions were slightly outdated. I have updated them now: it is also necessary to initialize the Rider frontend

**Comment by TimSirmovics** [comment]
Thanks for that.
Some more issues I came across:

On the 193 branch:
1. The `gradlew :prepare` command failed with an error. The RiderSdkPackageVersion.prop file was still generated so I moved on to the next step, but here is the error:
![image](https://user-images.githubusercontent.com/10243286/69678190-f9606780-10f8-11ea-817a-44f6703638fe.png)
2. In the last step of the build steps I have to run `./build Pack` instead of just `./build` to produce the .nupkg file.

I then realised the version number of the package did not match my R# version 2019.2.3, so I grabbed the net192 branch and tried to build, but I got a different error running `./gradlew :prepare`:
```
exception: java.lang.UnsupportedOperationException: This feature requires ASM7
        at org.jetbrains.org.objectweb.asm.ClassVisitor.visitNestMember(ClassVisitor.java:236)
        at org.jetbrains.org.objectweb.asm.ClassReader.accept(ClassReader.java:652)
        at org.jetbrains.org.objectweb.asm.ClassReader.accept(ClassReader.java:392)
        <long stack trace>
```
